examples/wesleycsj-helloWorld.cpp

```diff
@@ -0,0 +1,78 @@
+/*
+ *	Author: WesleyCSJ
+ *  Description: Render four colors on screen with 2s delay
+ *
+ *
+ */
+#include <iostream>
+#include <SDL2/SDL.h>
+#include "GameLoop.hpp"
+#include "InputHandler.hpp"
+
+#define SCREEN_WIDTH  800
+#define SCREEN_HEIGHT 600
+
+std::map<const char*, SDL_Keycode> input_config = {
+    {"left", SDLK_LEFT},
+    {"right", SDLK_RIGHT},
+    {"quit", SDLK_ESCAPE},
+    {"double_size", SDLK_SPACE},
+    {"run", SDLK_LSHIFT}
+};
+
+class Game: public GameLoop {
+	using GameLoop::GameLoop;
+	InputHandler input_handler = InputHandler(input_config);
+	float screenCounter = 0;
+	int colorCursor = 0;
+	int colors[4][4] = {
```

Comment:
Why are you changing the alpha channel in this example? To render a simple color there is no affect in a higher ou lower value.

Reply:
Because alpha is part of color component, in that yellow if you use 255 alpha instead the value it's there, it will become a yellowish green instead, not pure yellow.

Reply:
Are you sure? Did you tested in SDL?

Reply:
Well, it seems we need to activate the blending mode

Reply:
In theory it's always activated, but not in the mode that we expected. Since none is default and we don't need any other for this example, it will be not changed. Alpha property remains because there is this declaration for this function, meaning that even with NONE as blending mode, it should be provided.